src/windows/GeolocationProxy.js

```diff
@@ -16,7 +16,8 @@
 
```

Comment:
In ensureLocator you should also call requestAccessAsync:

"Starting in Windows 10 Insider Preview, call the RequestAccessAsync before accessing the user’s location. At that time, your app must be in the foreground and RequestAccessAsync must be called from the UI thread. Until the user grants your app permission to their location, your app can't access location data."

    var locator = new Windows.Devices.Geolocation.Geolocator();
    var loc;
    if (typeof Windows.Devices.Geolocation.Geolocator.prototype.requestAccessAsync === 'function')) {
        loc = locator.requestAccessAsync().then(function(result) { if (result) return locator; });
    } else { 
        loc = WinJS.Promise.wrap(locator);
    }